### PR TITLE
[Docs] Add InstructLab LLM guide to docs

### PR DESCRIPTION
**Pull request by kelbrown20**
**Description:** Adds detailed docs of the various LLMs to InstructLab repo 
Preview: https://github.com/kelbrown20/kelly-instructlab/blob/create-models-chart/docs/instructlab_models.md

### CODE REVIEW

**Comment by kelbrown20** [comment]
Requesting review but putting on hold because this PR needs to be updated after #2585 merges

**Review by jaideepr97 — APPROVED** [review]

**Pull request merged**
